docs/compute/auth/recoverPassword.md

```diff
@@ -0,0 +1,13 @@
+# Password Recovery
+
+If you forget your password click on **Forgot password** on the login page. Enter your email and click on the **Send 
+reset link** button.
+  <figure>
+    <a  href="../../../assets/images/auth/resetPassword.png">
+      <img width="400px" src="../../../assets/images/auth/resetPassword.png" alt="Recover password">
+    </a>
+    <figcaption>Password reset</figcaption>
+  </figure>
+
+A password reset link will be sent to the email address you entered if the email is found in the system.
+Click on the reset link to set your new password.
```

Comment:
Shouldn't there also be a page to recover second factor authentication?

Reply:
It is not there yet